dsl/src/main/kotlin/com/chutneytesting/kotlin/synchronize/ChutneyServerServiceImpl.kt

```diff
@@ -29,12 +37,38 @@ object ChutneyServerServiceImpl : ChutneyServerService {
         return HttpClient.get(serverInfo, "/api/scenario/component-edition/$scenarioId")
     }
 
-    override fun updateJsonScenario(serverInfo: ChutneyServerInfo, scenarioContent: String, scenarioId: String) {
+    override fun updateJsonScenario(serverInfo: ChutneyServerInfo, scenario: ChutneyScenario) {
+        var tags: List<String> = getScenarioTags(serverInfo, scenario)
+        val generatedTag = "GENERATED"
+        if (!tags.contains(generatedTag)) {
+            tags = tags.plus(generatedTag);
+        }
+        val body =
+            "{\"id\": ${SELENIUM_BY.id} , \"content\":\"${StringEscapeUtils.escapeJson(scenario.toString())}\", \"title\": \"${scenario.title}\", \"description\":\"${scenario.description}\" , \"tags\": ${
```

Comment:
Maybe use here [Kotlin raw string](https://kotlinlang.org/docs/strings.html#raw-strings) to have a cleaner JSON.
Same for the **createJsonScenario()** funtion.